### PR TITLE
[NETBEANS-54] Module Review

### PR DESCRIPTION
**Pull request by tomazst**
Missing Apache license headers are added.

### CODE REVIEW

File: jumpto/src/org/netbeans/modules/jumpto/file/FileSearchAction.java [hunk 1]
```diff
@@ -1,47 +1,20 @@
...
- * accompanied this code. If applicable, add the following below the
- * License Header, with the fields enclosed by brackets [] replaced by
- * your own identifying information:
- * "Portions Copyrighted [year] [name of copyright owner]"
- *
- * Contributor(s):
- *
- * The Original Software is the Accelerators module.
- * The Initial Developer of the Original Software is Andrei Badea.
- * Portions Copyright 2005-2006 Andrei Badea.
```
Comment (emilianbold) [inline jumpto/src/org/netbeans/modules/jumpto/file/FileSearchAction.java]: Hm, don't change this. We have to ask Andrei Badea for permission.

File: jumpto/src/org/netbeans/modules/jumpto/file/Bundle.properties [hunk 2]
```diff
@@ -1,45 +1,19 @@
...
-# accompanied this code. If applicable, add the following below the
-# License Header, with the fields enclosed by brackets [] replaced by
-# your own identifying information:
-# "Portions Copyrighted [year] [name of copyright owner]"
-#
-# Contributor(s):
-#
-# The Original Software is the Accelerators module.
-# The Initial Developer of the Original Software is Andrei Badea.
-# Portions Copyright 2005-2006 Andrei Badea.
```
Comment (emilianbold) [inline jumpto/src/org/netbeans/modules/jumpto/file/Bundle.properties]: Undo this one, have to see with Andrei Badea for permission.

File: jumpto/build.xml [hunk 3]
```diff
@@ -1,49 +1,21 @@
...
-accompanied this code. If applicable, add the following below the
-License Header, with the fields enclosed by brackets [] replaced by
-your own identifying information:
-"Portions Copyrighted [year] [name of copyright owner]"
-
-Contributor(s):
-
-The Original Software is the Accelerators module.
-The Initial Developer of the Original Software is Andrei Badea.
-Portions Copyright 2005-2006 Andrei Badea.
```
Comment (emilianbold) [inline jumpto/build.xml]: Undo this one, have to see with Andrei Badea for permission.

File: jumpto/src/org/netbeans/modules/jumpto/file/FileSearchOptions.java [hunk 4]
```diff
@@ -1,47 +1,20 @@
...
- * accompanied this code. If applicable, add the following below the
- * License Header, with the fields enclosed by brackets [] replaced by
- * your own identifying information:
- * "Portions Copyrighted [year] [name of copyright owner]"
- *
- * Contributor(s):
- *
- * The Original Software is the Accelerators module.
- * The Initial Developer of the Original Software is Andrei Badea.
- * Portions Copyright 2005-2006 Andrei Badea.
```
Comment (emilianbold) [inline jumpto/src/org/netbeans/modules/jumpto/file/FileSearchOptions.java]: Undo this one, have to see with Andrei Badea for permission.

File: jumpto/src/org/netbeans/modules/jumpto/resources/Bundle.properties [hunk 5]
```diff
@@ -1,45 +1,19 @@
...
-# accompanied this code. If applicable, add the following below the
-# License Header, with the fields enclosed by brackets [] replaced by
-# your own identifying information:
-# "Portions Copyrighted [year] [name of copyright owner]"
-#
-# Contributor(s):
-#
-# The Original Software is the Accelerators module.
-# The Initial Developer of the Original Software is Andrei Badea.
-# Portions Copyright 2005-2006 Andrei Badea.
```
Comment (emilianbold) [inline jumpto/src/org/netbeans/modules/jumpto/resources/Bundle.properties]: Undo this one, have to see with Andrei Badea for permission.

File: jumpto/src/org/netbeans/modules/jumpto/resources/layer.xml [hunk 6]
```diff
@@ -1,49 +1,21 @@
...
-accompanied this code. If applicable, add the following below the
-License Header, with the fields enclosed by brackets [] replaced by
-your own identifying information:
-"Portions Copyrighted [year] [name of copyright owner]"
-
-Contributor(s):
-
-The Original Software is the Accelerators module.
-The Initial Developer of the Original Software is Andrei Badea.
-Portions Copyright 2005-2006 Andrei Badea.
```
Comment (emilianbold) [inline jumpto/src/org/netbeans/modules/jumpto/resources/layer.xml]: Undo this one, have to see with Andrei Badea for permission.

File: jumpto/src/org/netbeans/modules/jumpto/type/GoToTypeAction.java [hunk 7]
```diff
@@ -1,47 +1,20 @@
...
- * by Oracle in the GPL Version 2 section of the License file that
- * accompanied this code. If applicable, add the following below the
- * License Header, with the fields enclosed by brackets [] replaced by
- * your own identifying information:
- * "Portions Copyrighted [year] [name of copyright owner]"
- *
- * Contributor(s):
- *
- * The Original Software is NetBeans. The Initial Developer of the Original
- * Software is Sun Microsystems, Inc. Portions Copyright 1997-2011 Sun
```
Comment (emilianbold) [inline jumpto/src/org/netbeans/modules/jumpto/type/GoToTypeAction.java:30]: This one is ok.

File: jumpto/test/unit/src/org/netbeans/modules/jumpto/common/HighlightingNameFormatterTest.java [hunk 8]
```diff
@@ -1,44 +1,20 @@
...
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
  *
- * Contributor(s):
- *   markiewb@netbeans.org
```
Comment (emilianbold) [inline jumpto/test/unit/src/org/netbeans/modules/jumpto/common/HighlightingNameFormatterTest.java]: Undo this one, have to see with markiewb for permission.

**Review by emilianbold — CHANGES_REQUESTED** [review]

File: jumpto/src/org/netbeans/modules/jumpto/file/FileSearchAction.java (hunk 1 above)
Comment (tomazst) [reply jumpto/src/org/netbeans/modules/jumpto/file/FileSearchAction.java]: Ok. I will put this four lines back.

Comment (tomazst) [reply jumpto/src/org/netbeans/modules/jumpto/file/FileSearchAction.java]: I must put back this 5 lines in the license header. OK?
* Contributor(s):
*
* The Original Software is the Accelerators module.
* The Initial Developer of the Original Software is Andrei Badea.
* Portions Copyright 2005-2006 Andrei Badea.

Comment (emilianbold) [reply jumpto/src/org/netbeans/modules/jumpto/file/FileSearchAction.java]: No, revert the change on the whole file until we figure out what to do with it on the dev@ mailing list.

**Comment by emilianbold** [comment]
I believe these two files could have been updated:

jumpto/apichanges.xml
jumpto/src/org/netbeans/modules/jumpto/type/GoToTypeAction.java

you only updated one of them now.

**Pull request merged**